src/rapids_dependency_file_generator/cli.py

```diff
@@ -16,6 +17,11 @@ def validate_args(argv):
         default=default_dependency_file_path,
         help="Path to YAML config file",
     )
+    parser.add_argument(
+        "--validate",
+        action="store_true",
+        help="Validate the config file in full before generating anything.",
+    )
```

Comment:
It might make sense to always validate instead of requiring a flag.

Reply:
Done.